stories/multi-combo-box/multi-combo-box.stories.js

```diff
@@ -0,0 +1,929 @@
+export default {
+    title: 'Patterns/Multi ComboBox',
+    parameters: {
+        description: `
+The multi-combo-box component is an opinionated composition of the \`input group\`, \`popover\`, \`checkbox\`, \`list\` and \`token\` components.
+The \`popover\` is shared between the combobox and select components. Please see the \`select\` documentation for the complete list of modifiers,
+That are also  supported by the \`combobox\` component.
+
+\`Multi ComboBox\` allows users to enter multiple values which are displayed as a tokens. 
+It provides an editable input field for filtering the list, and a dropdown menu with a list of the available options. 
+If the entries are not validated by the application, users can also enter custom values. 
+`,
+        tags: ['f3', 'a11y', 'theme'],
+        components: [
+            'form-label',
+            'popover',
+            'radio',
+            'title',
+            'token',
+            'input-group',
+            'checkbox',
+            'list',
+            'button',
+            'icon',
+            'tokenizer',
+            'dialog',
+            'bar',
+            'input',
+            'layout-grid'
+        ]
+    }
+};
+
+export const cozyAndCompact = () => `<div class="fd-container">
+    <div class="fd-row">
+	<div class="fd-col fd-col--6">
+        <label class="fd-form-label">
+            Multi ComboBox Cozy Mode
+        </label>
+        <div class="fd-popover">
+            <div class="fd-popover__control" aria-controls="F4GcX348a" aria-expanded="true" aria-haspopup="true">
```

Comment:
Slight correction to my previous review, this is the element that needs to contain the `combobox` attributes. You can remove the ones from the `input`s. Refer to https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html
```suggestion
            <div class="fd-popover__control" role="combobox" aria-controls="F4GcX348a" aria-expanded="true" aria-haspopup="true">
```